File: klasa 2/python/funkcje.py
#!/usr/bin/env python
# -*- coding: utf-8 -*-
#
#  wejscie-wyjscie.py
# duck typing
# zasieg zmiennych: zasieg lokalny (ograniczony np. funkcją)

def suma(a, b):
    return a + b
    
def roznica(a, b):
    return a - b
    
def iloczyn(a, b):
    return a * b
    
def iloraz(a, b):
    return a / b
    
    

def main(args):
    a = int(input("Podaj pierwsza liczbe: "))
    print(a)
    
    b = int(input("Podaj druga liczbe: "))
    print(b)
    
    print("Suma: ", suma(a, b))
    print("Różnica: ", roznica(a, b))
    print("Iloczyn: ", iloczyn(a, b))
    print("Iloraz: ", iloraz(a, b))
    
    
    return 0

if __name__ == '__main__':
    import sys
    sys.exit(main(sys.argv))

File: Kody-master/klasa 2/cpp/funkcje.cpp
#include <iostream>

using namespace std;

int suma(int a, int b){
    return a + b;
    }

int roznica(int a, int b){
    return a - b;
    }

int iloraz(int a, int b){
    return a / b;
    }

int iloczyn(int a, int b){
    return a * b;
    }



int main(int argc, char **argv)
{
	int a, b;
    a = b = 0;
    
    cout << "Podaj pierwsza liczbe: ";
    cin >> a;
    cout << a;
      
    cout << endl << "Podaj druga liczbe: ";
    cin >> b;
    cout << b;
    
    cout << endl << "Suma: " << suma(a, b);
    cout << endl << "Różnica: " << roznica(a, b);
    cout << endl << "Iloraz: " << iloraz(a, b);
    cout << endl << "Iloczyn: " << iloczyn(a, b);
    
	return 0;
}
